tests/test_transf_reif.py

```diff
@@ -12,26 +12,6 @@ def setUp(self):
         _IntVarImpl.counter = 0
         _BoolVarImpl.counter = 0
 
-    def test_only_bv_implies(self):
```

Comment:
These tests should remain in some form or another, but probably this will get split into another pull request anyway